enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -138,5 +138,62 @@ export function getEmbeddingComponentOverrides(
         ...(theme?.popover?.zIndex && { zIndex: theme.popover.zIndex }),
       },
     },
+    // TODO: fix those, they're not all correct
+    ModalRoot: {
```

Comment:
I copy pasted the default props for all the components, but as you can probably tell some of them are broken (i'm passing target both as first level and nested in portalProps, I just wanted to have something i could test locally, I'll manually check which one are needed before we merge this